src/content/docs/apis/nerdgraph/examples/nerdgraph-data-partition-rules-tutorial.mdx

```diff
@@ -64,7 +63,19 @@ Available data partition rule fields include:
       </td>
 
       <td>
-        The matching criteria for this data partition rule. Once the rule is enabled, logs matching this criteria will be routed to the specified data partition.
+         <i>(deprecated)</i> The matching criteria for this data partition rule. Once the rule is enabled, logs matching this criteria will be routed to the specified data partition. 
+        This field has been deprecated and will eventually be replaced with the `nrql` field to allow for more flexibility when creating data partitions. If the rule is
+        specified with `nrql` that cannot be parsed into `matchingCriteria`, this field will be null.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        NRQL
+      </td>
+
+      <td>
+        The matching criteria for this data partition rule specified using a NRQL `where` clause. Once the rule is enabled, logs matching this criteria will be routed to the specified data partition.
```

Comment:
Maybe uppercase that WHERE to distinguish it from the other things we're putting in sans-serif font?

Also consider linking to the NRQL docs: https://docs.newrelic.com/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/